bootstrap/pkg/kfupgrade/kfupgrade.go

```diff
@@ -216,17 +216,21 @@ func MergeKfDef(oldKfDef *kfdefsv3.KfDef, newKfDef *kfdefsv3.KfDef) {
 }
 
 func (upgrader *KfUpgrader) Generate() error {
-	kfApp := kustomize.GetKfApp(upgrader.NewKfDef)
-	return kfApp.Generate(kftypesv3.K8S)
+	// TODO: fix this
```

Comment:
Why is this commented out?

Reply:
@richardsliu Is this a bug in kfupgrader? Why are we calling kustomize.GetKFApp?

Reply:
I need to either migrate kfupgrade to use kfconfig as well in this PR, or in a followup PR.
WDYT?

Reply:
Chatted with @richardsliu He's going to update KfUpgrader. I think what we discussed is

* kfupgrader will be updated to store the filepath of the KFDef
* We will call coordinator.
https://github.com/kubeflow/kubeflow/blob/9f5ef66ce3d12ff2b603d07330a16661caa0b286/bootstrap/pkg/kfapp/coordinator/coordinator.go#L220

Reply:
so ok if I leave it as commented out here?

Reply:
I think so.